src/build/__main__.py

```diff
@@ -365,9 +369,9 @@ def main(cli_args: Sequence[str], prog: Optional[str] = None) -> None:  # noqa:
                 args.srcdir, outdir, distributions, config_settings, not args.no_isolation, args.skip_dependency_check
             )
             artifact_list = _natural_language_list([f'[underline]{artifact}[reset][bold][green]' for artifact in built])
-            _print(f'[bold][green]Successfully built {artifact_list}')
+            _print(f'[bold][green]Successfully built {artifact_list}[reset]')
     except Exception as e:  # pragma: no cover
-        _print('\n[dim]{}[reset]\n'.format(traceback.format_exc().strip('\n')))
+        print(_format('\n[dim]{}[reset]\n').format(traceback.format_exc().strip('\n')))
```

Comment:
> a bit more complexity than what we need

How is this not complex? In six months, will you remember the difference between `print`, `_print`, `format`, and `_format`?

I'd do either:

```python
        print('\n{dim}{msg}{reset}\n').format(msg=traceback.format_exc().strip('\n'), **_STYLES)
```

And remove `_print` and `_format` entirely. If NO_COLOR is specified, the dict would just be empty strings.

Or at the very least have `_format` pass-through to format, so you could do this:

```python
        print(_format('\n[dim]{}[reset]\n', traceback.format_exc().strip('\n'))
```

Reply:
I changed to

```python
print('{red}ERROR{reset} {}'.format(msg, **_STYLES))
print('\n{dim}{}{reset}\n'.format(tb.strip('\n'), **_STYLES))
...
```